OrbitGl/TimeGraph.cpp

```diff
@@ -872,10 +872,10 @@ void TimeGraph::SortTracks() {
     m_EventCount.clear();
 
     for (auto& pair : GEventTracer.GetEventBuffer().GetCallstacks()) {
-      ThreadID threadID = pair.first;
-      std::map<uint64_t, CallstackEvent>& callstacks = pair.second;
-      m_EventCount[threadID] = callstacks.size();
-      GetOrCreateThreadTrack(threadID);
+      ThreadID thread_id = pair.first;
```

Comment:
I saw that you replaced ThreadID by uint32_t earlier, do you want to do it here as well?

Reply:
Done.